servers/src/main/java/tachyon/master/next/block/meta/MasterBlockInfo.java

```diff
@@ -57,12 +57,12 @@ public synchronized void addWorker(long workerId, int tierAlias) {
     mWorkerIdToAlias.put(workerId, tierAlias);
   }
 
-  public void removeWorker(long workerId) {
-    mWorkerIdToAlias.remove(workerId);
+  public int removeWorker(long workerId) {
+    return mWorkerIdToAlias.remove(workerId);
```

Comment:
Good catch. It will throw a NPE, because the hashmap will return null, and the unboxing to int will result in the exception. I think we can return Integer instead?